tests/cephadm/test_admin_node_removal.py

```diff
@@ -0,0 +1,20 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+
+    config = kw.get("config")
+    # Removing _admin node from bootstrp node
+    instance = CephAdmin(cluster=ceph_cluster, **config)
+    admin_node = instance.cluster.get_nodes()[0].hostname
+    cmd = "cephadm shell ceph orch host rm {}".format(admin_node)
+    try:
+        out, err = instance.installer.exec_command(cmd, sudo=True)
+        log.error("Test case failed, Admin node removed")
+        return 1
+    except Exception:
```

Comment:
Don't use general exception in this case. Check for specific exception and error message

Reply:
Updated